### PR TITLE
Update plotly docs

### PR DESCRIPTION
**Pull request by rashidakanchwala**
## Description

Just clarified which Kedro dataset supports which plotly packages. 

## Checklist

- [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro/pull/1557"></a>

### CODE REVIEW

**Review by AhdraMeraliQB — APPROVED** [review]

**Review by SajidAlamQB — APPROVED** [review]
Awesome! 💥

File: docs/source/tutorial/visualise_pipeline.md [hunk 1]
```diff
@@ -164,7 +160,7 @@ shuttle_passenger_capacity_plot:
 ```
 
 
-- [plotly.JSONDataSet](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.plotly.JSONDataSet.html#kedro.extras.datasets.plotly.JSONDataSet) - To use this dataset you need to configure your plot using plotly-express python library in your kedro node.
+- [plotly.JSONDataSet](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.plotly.JSONDataSet.html#kedro.extras.datasets.plotly.JSONDataSet) - To use this dataset you need to configure your plot using plotly-express python library in your kedro node. This dataset supports both [Plotly Express](https://plotly.com/python/plotly-express) and [Plotly Graph Objects](https://plotly.com/python/graph-objects/).
```
Comment (yetudada) [inline docs/source/tutorial/visualise_pipeline.md:163]:
```suggestion
- [plotly.JSONDataSet](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.plotly.JSONDataSet.html#kedro.extras.datasets.plotly.JSONDataSet) - To use this dataset you need to configure your plot using the `plotly-express` Python API in your Kedro node. This dataset supports [Plotly Express](https://plotly.com/python/plotly-express) and [Plotly Graph Objects](https://plotly.com/python/graph-objects/).
```

Comment (yetudada) [reply docs/source/tutorial/visualise_pipeline.md:163]: How do users configure Plotly Graph Objects? And do we give them an example?

File: docs/source/tutorial/visualise_pipeline.md [hunk 2]
```diff
@@ -118,7 +114,7 @@ You need to update `requirements.txt` in your Kedro project and add the followin
 You can view Plotly charts in Kedro-Viz when you use Kedro's plotly datasets.
 
 There are two types of plotly datasets in Kedro :
-- [plotly.PlotlyDataSet](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.plotly.PlotlyDataSet.html#kedro.extras.datasets.plotly.PlotlyDataSet) - To use this dataset you need to configure your plot in the `catalog.yml`.
+- [plotly.PlotlyDataSet](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.plotly.PlotlyDataSet.html#kedro.extras.datasets.plotly.PlotlyDataSet) - To use this dataset you need to configure your plot in the `catalog.yml`. This datasets supports only [Plotly Express](https://plotly.com/python/plotly-express).
```
Comment (yetudada) [inline docs/source/tutorial/visualise_pipeline.md:117]:
```suggestion
- [plotly.PlotlyDataSet](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.plotly.PlotlyDataSet.html#kedro.extras.datasets.plotly.PlotlyDataSet) - To use this dataset you need to configure your plot in the `catalog.yml`. This datasets only supports [Plotly Express](https://plotly.com/python/plotly-express).
```

[the remark this answers is not in the document]
Comment (yetudada) [reply docs/source/tutorial/visualise_pipeline.md:117]: I don't think we need to mention this. But there's a point around telling users what they can do with each Plotly version. Perhaps we can add something like, "This datasets supports only [Plotly Express](https://plotly.com/python/plotly-express) which allows you to create basic charts like scatter plots or bar and pie charts."

**Review by yetudada — COMMENTED** [review]
Left some minor comments and improvements

File: docs/source/tutorial/visualise_pipeline.md (hunk 1 above)
Comment (rashidakanchwala) [reply docs/source/tutorial/visualise_pipeline.md:163]: basically, if you use JsonDataSet you can do as you like, create a chart using PlotlyGraphObjects or PlotlyExpress. There's no configuration required like plotly.PlotlyDataset which uses Plotly Express specifically under the hood

File: docs/source/tutorial/visualise_pipeline.md (hunk 2 above)
Comment (MerelTheisenQB) [inline docs/source/tutorial/visualise_pipeline.md:117]:
```suggestion
- [plotly.PlotlyDataSet](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.plotly.PlotlyDataSet.html#kedro.extras.datasets.plotly.PlotlyDataSet) - To use this dataset you need to configure your plot in the `catalog.yml`. This dataset only supports [Plotly Express](https://plotly.com/python/plotly-express).
```

**Review by MerelTheisenQB — APPROVED** [review]
LGTM 👍

File: docs/source/tutorial/visualise_pipeline.md [hunk 3]
```diff
@@ -164,18 +160,18 @@ shuttle_passenger_capacity_plot:
 ```
 
 
-- [plotly.JSONDataSet](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.plotly.JSONDataSet.html#kedro.extras.datasets.plotly.JSONDataSet) - To use this dataset you need to configure your plot using plotly-express python library in your kedro node.
+- [plotly.JSONDataSet](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.plotly.JSONDataSet.html#kedro.extras.datasets.plotly.JSONDataSet) - To use this dataset you need to configure your plot in your kedro node. This dataset supports both [Plotly Express](https://plotly.com/python/plotly-express) and [Plotly Graph Objects](https://plotly.com/python/graph-objects/).
 
 
-Below is an example of how to visualise plots on Kedro-Viz using `plotly.JSONDataSet`
+Below is an example of how to visualise plots using both [Plotly Express](https://plotly.com/python/plotly-express) and [Plotly Graph Objects](https://plotly.com/python/graph-objects/) on Kedro-Viz using `plotly.JSONDataSet`
```
Comment (yetudada) [inline docs/source/tutorial/visualise_pipeline.md:166]:
```suggestion
Below is an example of how to visualise plots using [Plotly Express](https://plotly.com/python/plotly-express) and [Plotly Graph Objects](https://plotly.com/python/graph-objects/) on Kedro-Viz using the `plotly.JSONDataSet`.
```

File: docs/source/tutorial/visualise_pipeline.md [hunk 4]
```diff
@@ -118,7 +114,7 @@ You need to update `requirements.txt` in your Kedro project and add the followin
 You can view Plotly charts in Kedro-Viz when you use Kedro's plotly datasets.
 
 There are two types of plotly datasets in Kedro :
```
Comment (yetudada) [inline docs/source/tutorial/visualise_pipeline.md:116]:
```suggestion
There are two types of Plotly datasets in Kedro, the `plotly.PlotlyDataSet` and `plotly.JSONDataSet`. 
```

File: docs/source/tutorial/visualise_pipeline.md [hunk 5]
```diff
@@ -164,18 +160,18 @@ shuttle_passenger_capacity_plot:
 ```
 
 
-- [plotly.JSONDataSet](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.plotly.JSONDataSet.html#kedro.extras.datasets.plotly.JSONDataSet) - To use this dataset you need to configure your plot using plotly-express python library in your kedro node.
+- [plotly.JSONDataSet](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.plotly.JSONDataSet.html#kedro.extras.datasets.plotly.JSONDataSet) - To use this dataset you need to configure your plot in your kedro node. This dataset supports both [Plotly Express](https://plotly.com/python/plotly-express) and [Plotly Graph Objects](https://plotly.com/python/graph-objects/).
```
Comment (yetudada) [inline docs/source/tutorial/visualise_pipeline.md:163]:
```suggestion
### [plotly.JSONDataSet](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.plotly.JSONDataSet.html#kedro.extras.datasets.plotly.JSONDataSet)

To use this dataset you need to configure your plot in your Kedro node. This dataset supports [Plotly Express](https://plotly.com/python/plotly-express) and [Plotly Graph Objects](https://plotly.com/python/graph-objects/).
```

**Review by yetudada — APPROVED** [review]
I left some minor improvements. Most of the changes are just around formatting.

**Pull request merged**